src/__init__.py

```diff
@@ -294,10 +296,11 @@ def generate_basic(self):
 
 
         after_generate_basic(self, self.multiworld, self.player)
-        # Uncomment these to generate a diagram of your manual.  Only works on 0.4.4+
 
-        # from Utils import visualize_regions
-        # visualize_regions(self.multiworld.get_region("Menu", self.player), f"{self.game}_{self.player}.puml")
+        # Enable this in Meta.json to generate a diagram of your manual.  Only works on 0.4.4+
+        if enable_puml:
```

Comment:
I love this, because I've definitely found myself wanting an option I could turn on to make the UML. I'm torn on whether it should be a YAML option or a JSON setting, though.

Might name it something more obvious like "region_diagram" instead of "puml", for the less technical folks in the community. 😃

Reply:
I moved it into meta.py so a hook could change it if someone want to enable but that way.
ill rename it quickly.